terraform/acm.tf

```diff
@@ -3,6 +3,11 @@ resource "aws_acm_certificate" "apps" {
   validation_method = "DNS"
 }
 
+resource "aws_acm_certificate" "non-prod" {
+  domain_name = "*.apps.${local.non-prod_domain_name}"
```

Comment:
mixing `-` and `_` in a variable name is a bit weird - can you change it to `non_prod` or `nonprod`?

Reply:
no probs - now changed.